src/server/api/routes/api-router.js

```diff
@@ -4,6 +4,7 @@ const router = express.Router();
 
 // Router imports
 const modulesRouter = require('./modules.router');
```

Comment:
You can remove code related to modules, after you created  for another endpoint since it was just for sample/template,

Reply:
@BikramRawat you cannot remove something that belongs to another feature via a PR for a different feature

Reply:
Please remove anything related to modules. Remember all of the references in the router file, etc. Make sure it runs after you remove the files.